crates/ethjson/src/state/test.rs

```diff
@@ -86,16 +88,27 @@ pub struct MultiTransaction {
 
 impl MultiTransaction {
     /// Build transaction with given indexes.
-    pub fn select(&self, indexes: &PostStateIndexes) -> Transaction {
-        Transaction {
+    pub fn select(&self, indexes: &PostStateIndexes) -> TypedTransaction {
```

Comment:
hm if I saw this correctly select is only used to return Transaction and then only to transfer it to SignedTransaction.
Maybe we can remove this Transaction and use SignedTransaction from ethcore this would simplify and remove the doubling of structures that we use.

Reply:
Nice suggestion. Managed to decouple ethcore/types/ from ethjson and eliminate the duplicate structures.